src/net/sourceforge/kolmafia/request/RelayRequest.java

```diff
@@ -733,6 +733,18 @@ private void sendLocalFile(final String filename) {
     } catch (IOException e) {
     }
 
+    // If it's a binary file, send it back without loading it as a string.
+    if (!this.contentType.startsWith("text/") && !this.contentType.startsWith("application/")) {
```

Comment:
assuming the latter is to catch `application/json`, might be nicer to just consistently use `text/json`? e.g. `application/octet-stream` should also be sent as bytes